pkg/util/graph/backup_graph.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of the Kubevirt Velero Plugin project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Velero Plugin Authors.
+ *
+ */
+
+package vmgraph
```

Comment:
@mhenriks wondering if `vmgraph` is good name for the package since it also supports other objects such as DVs.

Reply:
Yeah `objgraph` or something more generic is probably better

Curious why you chose a different package name (vmgraph) and directory (graph)?

Reply:
I though `vmgraph` was better for an exportable package as sounded less generic while `graph` worked in the context of our directories, but yeah probably not the best naming decision. I'll change the package name.

Reply:
What about `kvgraph`?

Reply:
Renamed to `kvgraph`